cmake/opentelemetry-proto.cmake

```diff
@@ -328,12 +328,10 @@ add_library(
 set_target_version(opentelemetry_proto)
 
 # Disable include-what-you-use on generated code.
-set_target_properties(
-  opentelemetry_proto
-  PROPERTIES CXX_INCLUDE_WHAT_YOU_USE ""
-)
+set_target_properties(opentelemetry_proto PROPERTIES CXX_INCLUDE_WHAT_YOU_USE
```

Comment:
Is this formatting only?

Reply:
Yes, this is formated by cmake-format , base on `.cmake-format.py`